include/librealsense2/dds/topics/device-info/device-info-msg.h

```diff
@@ -0,0 +1,38 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include <string>
+#include "deviceInfoPubSubTypes.h"
+
+namespace librealsense {
+namespace dds {
+namespace topics {
+struct device_info
+{
+    std::string name;
+    std::string serial;
+    std::string product_line;
+    bool locked;
+
+    device_info()
+        : name()
+        , serial()
+        , product_line()
+        , locked( false )
+    {
+    }
+
+    device_info( const topics::raw::device_info & dev )
```

Comment:
Don't need `topics::` here, do we? Is there a purpose for it here?

Reply:
No need, will remove it.